Backtracking/KnightTour.js

```diff
@@ -22,37 +22,37 @@ class OpenKnightTour {
     return moves.filter(([y, x]) => y >= 0 && y < this.size && x >= 0 && x < this.size)
   }
 
-  isComplete () {
+  isComplete() {
     // helper function to check if the board is complete
     return !this.board.map(row => row.includes(0)).includes(true)
   }
 
-  solve () {
+  solve() {
     // function to find the solution for the given board
     for (let i = 0; i < this.size; i++) {
       for (let j = 0; j < this.size; j++) {
-        if (this.solveHelper([i, j], 0)) return true
+        return this.solveHelper([i, j], 0)
```

Comment:
This is wrong. The previous code would only return early if this was truthy. Now it will return early as soon as this returns, even if it's falsy.